sql/updates/world/2015_05_26_07_world.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
```

Comment:
INSERT INTO queries need a DELETE FROM query first to avoid errors. Insert the following line as a new line 1:
```sql
DELETE FROM `spell_script_names` WHERE `spell_id`= 45724 AND `ScriptName`= 'spell_midsummer_braziers_hit';
```
This will make it possible to run the SQL more than one time and avoid errors on subsequent executions.